### PR TITLE
content: write description for project heading in project page

### LINKED ISSUE #85: Content: Add description for project in Projects page
### **DESCRIPTION**
write the description for the project heading

### **Mocks**

image\

### Acceptance Criteria
**Update [Required]**
- [x] projects content

**Definition of Done**

- [x]  All of the required items are completed.
- [x]  Approval by 1 mentor.

**Estimation**
3 hours

### PR DESCRIPTION
**Pull request by Souvikns**
### Description

Created Project tab using flex property. 

Fixes #85 

### Type of Change:
**Delete irrelevant options.**
- Documentation

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### Checklist:
**Delete irrelevant options.**
- [x] My PR follows the style guidelines of this project
- [x] I have made corresponding changes to the documentation

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by nandini45** [comment]
no issue about the image i suppose but i think i should ask Anna first
  @annabauza
by the way you can add more content to it.. 
here i made a change. you can edit it or try something like that

Anita Borg the famous computer scientist said,"We'll try to be a catalyst for development projects that include women at all levels - in the design and development process."
     At AnitaB.org, we envision a future where the people who imagine and build technology mirror the people and societies for whom they build it. We connect, inspire, and guide women in computing, and organizations that view technology innovation as a strategic imperative. 
                     Our team under the guidance of our admins is working together to make some amazing projects. Scroll down to know about it and if interested start contributing to them.

you have to add more content i gave a idea

**Comment by nandini45** [comment]
@Souvikns  can u firstly edit your pull request template. if you want i will edit it but remember the template and accordingly you have to make changes in future

**Comment by Souvikns** [comment]
Ok, I will be adding more content, also in The mockup in the issue #86 there were cards, I can pull all the details of the repositories of this org and display in cards, @nandini45 let me know if you want that.

**Comment by nandini45** [comment]
no actually for the details of the card there is a separate issue.. and i think gauri is working on it. just the heading for now.. thanks
if you are looking for more contribution in  content you can write content for our medium blog

**Comment by Souvikns** [comment]
I changed the content as you have said, I don't think there is anything to add though. It is pretty much complete.

**Comment by nandini45** [comment]
ok i will wait for the final review from Anna and then it will be done

File: .gitignore
Comment (annabauza) [inline .gitignore:7]: remove this line please

File: src/Components/Projects/index.js
Comment (annabauza) [inline src/Components/Projects/index.js:20]: we don't use `rem` as this will break the mobile native

File: src/Components/Projects/index.js
Comment (annabauza) [inline src/Components/Projects/index.js:33]: we don't use `rem` as this will break the mobile native

File: src/Components/Projects/index.js
Comment (annabauza) [inline src/Components/Projects/index.js:37]: we don't use `rem` as this will break the mobile native

File: src/assets/project.svg
Comment (annabauza) [inline src/assets/project.svg:1]: we use pngs

File: src/Components/Projects/index.js
Comment (annabauza) [inline src/Components/Projects/index.js:19]: style should be defined outside of component and referred here.

File: src/Components/Projects/index.js
Comment (annabauza) [inline src/Components/Projects/index.js:28]: style should be defined outside of component and referred here.

File: src/Components/Projects/index.js
Comment (annabauza) [inline src/Components/Projects/index.js:33]: style should be defined outside of component and referred here.

File: src/Components/Projects/index.js
Comment (annabauza) [inline src/Components/Projects/index.js:37]: style should be defined outside of component and referred here.

File: src/Components/Projects/index.js
Comment (annabauza) [inline src/Components/Projects/index.js:39]: Text should be kept outside of view in json stucture.

File: src/Components/Projects/index.js
Comment (annabauza) [inline src/Components/Projects/index.js:50]: style should be defined outside of component and referred here.

File: src/Components/Projects/index.js
Comment (annabauza) [inline src/Components/Projects/index.js:53]: style should be defined outside of component and referred here.

File: src/Components/Projects/index.js
Comment (annabauza) [inline src/Components/Projects/index.js:54]: please have a look how images are loaded in sponsors and follow the pattern.

File: src/Components/Projects/index.js
Comment (Souvikns) [reply src/Components/Projects/index.js:20]: I come from react background that's why i have a tendency of using rem, i will change to % and aspect ratio

**Comment by Souvikns** [comment]
I have shifted all the styling in one object and also replaced the rems accordingly, Currently I have shifted all the contents inside a object and kept in the same file because I feared moving it in the content folder I might break something, 

About the images I am not sure what the best practice for putting images, thus I tried to refer to the existing code,
```
<Image
    style={{ height: 50, width: 100 }}
    source={require('./../../assets/logo.png')}
/>
```
This is how the image was written in the header component, and I had followed that, I am having problem with understanding the image, issue.

**Comment by annabauza** [comment]
@Souvikns  this is content PR not coding, right?

**Comment by Souvikns** [comment]
Yeah it was content PR but i the problem statement also required to code, thats why I applied.

**Review by annabauza — CHANGES_REQUESTED** [review]
This PR should be only description - it is not a coding task.

**Comment by nandini45** [comment]
@Souvikns  can you update your work as required?

**Comment by Souvikns** [comment]
I made all the necessary changes, but as @annabauza said-
> This PR should be only description - it is not a coding task.

I did change the code according to the review. but I think this issue is something different.

**Comment by annabauza** [comment]
Closing this PR because it is not content only. Please open new PR - content only. Also when you do coding issue, please look at the standards of coding, react-native orientation so we could migrate components to native app. Thank you.

**Pull request closed without merging**
